docs/index.rst

```diff
@@ -1,47 +1,98 @@
-.. OpenTelemetry documentation master file, created by
-   sphinx-quickstart on Mon Jun  3 22:48:38 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+OpenTelemetry Python
+====================
 
-OpenTelemetry
-=============
-
-Welcome to OpenTelemetry's documentation!
+Welcome to OpenTelemetry Python documentation!
 
 This documentation describes the ``opentelemetry-api``, ``opentelemetry-sdk``
 and integration packages.
 
-.. toctree::
-   :maxdepth: 1
-   :caption: OpenTelemetry API:
+TODO: intro with link to opentelemetry.io
```

Comment:
can we remove TODOs, or use sphinx comments? 

I'd say the omission of data isn't a big detail, we can always file subsequent PRs to fill that in.